fastlane/Fastfile

```diff
@@ -88,7 +88,7 @@ platform :android do
     app_name = options[:app_name]
 
     generic_sentry_upload(
-        package_name: "tuerantuer.app.integreat", # FIXME
+        package_name: "tuerantuer.app.integreat", # FIXME: bundle id
```

Comment:
why is there the fixme for the bundle id? do we only get this id if we have had our "first" release?

Reply:
I'm currently resolving this in my current task. We already know the id, but yes, this will only be used as soon as we do the first release.